monkey/agent_plugins/payloads/cryptominer/manifest.yaml

```diff
@@ -0,0 +1,19 @@
+name: Cryptominer
+plugin_type: Payload
+supported_operating_systems:
+  - linux
+  - windows
+target_operating_systems:
+  - linux
+  - windows
```

Comment:
We can omit these for payloads. Though I don't think this will hurt anything.

Reply:
I'd rather be explicit about it.